app/assets/javascripts/discourse/tests/acceptance/pending-posts-test.js

```diff
@@ -0,0 +1,31 @@
+import { acceptance } from "discourse/tests/helpers/qunit-helpers";
+import { test } from "qunit";
+import { click, visit } from "@ember/test-helpers";
+import { setupApplicationTest as EMBER_CLI_ENV } from "ember-qunit";
+
+acceptance("Pending posts - no existing pending posts", function (needs) {
+  if (!EMBER_CLI_ENV) {
```

Comment:
Had to do this at several places as we did for sticky avatars. Otherwise some of the newest helpers aren’t available with the legacy env and tests are just failing. I guess it’s ok as they’re working just fine in Ember CLI jobs.

Reply:
Yep I think it's fine, will be easy to grep over them and remove this once we are full on ember-cli

Reply:
Agreed!